cloud/main.js

```diff
@@ -33,6 +33,65 @@ Parse.Cloud.define("sendMessage", function(request, response) {
                        });
 });
 
+Parse.Cloud.define("registerNewUser", function(request, response) {
+    getUserCount(function(userCount) {
+        if (userCount == undefined) {
+            response.error;
```

Comment:
You should call return here so the rest of the function doesn't run. 

Also this should response.error();